src/github3/orgs.py

```diff
@@ -1461,3 +1524,46 @@ def edit(self, state):
             self._update_attributes(json)
             return True
         return False
+
+
+class OrganizationHook(hook.Hook):
+    """The representation of a hook on an organization.
+
+    See also: https://developer.github.com/v3/orgs/hooks/
+
+    This object has the same attributes as :class:`~github3.repos.hook.Hook`.
+    It overrides method :meth:`edit`, since organization and repository hooks
+    have different signatures.
+    """
+
+    class_name = "OrganizationHook"
```

Comment:
This doesn't seem to be used, and can probably just disappear

Reply:
Done